docs/08-logo.md

```diff
@@ -0,0 +1,70 @@
+Een logo is een beeldmerk dat wordt ingezet voor de herkenbaarheid van de organisatie in allerlei communicatie-uitingen. Een logo kan een grafisch element zijn of bestaan uit tekst of een combinatie van beiden. Een logo is een belangrijk onderdeel van de huisstijl en vormt de identiteit van de organisatie.
+
+## Hoe te gebruiken: formaten en afmetingen
+
+<table>
+    <tr>
+        <td width='33%'><h3>Logo met extra tekstregel</h3></td>
+    </tr>
+    <tr>
+        <td valign="top">De extra tekstregel onder het logo is in het font Asap regular 20pt en in grijs (#666666)<br><br>
+
+De afstand tussen ‘Omgevingsloket’ en de ‘Extra tekstregel’ is 36px<br><br>
+
+De tekstregel lijnt links aan de ‘O’ van 'Omgevingsloket'</td>
+        <td width='66%'>![Logo's](/logos/logogebruik_omgevingsloket.svg "Logos")</td>
+    </tr>
+</table>
+
+* Het logo wordt altijd linksboven op de website geplaatst
+* De breedte van het logo 'Omgevingsloket' is 256 pixels (3 koloms bij een breedte van 1440 pixels)
+* Als de tekstregel breder is dan het logo van het Omgevingsloket loopt deze door maar dan is de breedte van het logo Omgevingsloket zelf, nog steeds 256 pixels. Dit is om eenheid te behouden en te voorkomen dat logo's gaan springen bij wisselen van pagina's. De extra tekstregel loopt dan breder uit dan de 256 pixels van het logo
+* Het logo is beschikbaar in het SVG format zodat makkelijk geschaald kan worden naar andere formaten zonder het logo weer opnieuw te moeten aanpassen
+
+## Hoe te gebruiken: logo en typografie
+
+<table>
+    <tr>
+        <td width='33%'><h3>Webfont Asap</h3></td>
+    </tr>
+    <tr>
+        <td valign="top">
+        Het webfont “Asap” is onderdeel van de vrij beschikbare Google Fonts. Dit webfont is optimaal voor schermweergave. <br><br>
+
+        Asap heeft als eigenschap dat alle karakters in het lettertype een gedeelde breedte hebben. <br><br>
+
+        De lopende tekst blijft er voor de eindgebruiker rustig uitzien, ook bij veranderingen van de stijl van het lettertype.</td>
+        <td width='66%'>![Typografie](/logos/logogebruik_typografie.svg "Typografie")</td>
```

Comment:
Hoe staat dit in verhouding tot de al bestaande pagina "Typography": https://dso-toolkit.nl/9.0.0/docs/typography.html

En waarom schrijf je deze letters niet uit zoals Google Fonts dat doet? https://fonts.google.com/specimen/Asap